repose-aggregator/src/docs/asciidoc/filters/herp.adoc

```diff
@@ -1,3 +1,213 @@
-= Highly Efficient Record Processor Filter
+= Highly Efficient Record Processor (HERP) Filter
 
 include::../_includes/in-progress.adoc[]
+
+The Highly Efficient Record Processor (HERP) filter logs an event for each API request.
+These logs provide information regarding the processing of each request.
+The logs can be inspected to view the usage of a service and to perform auditing.
+Events are recorded even if the API request is rejected by Repose before reaching the origin service.
+
+== General filter information
+* *Filter name:* HERP
+* *Filter configuration:* highly-efficient-record-processor.cfg.xml
+* *Released:* v7.0.0.1
+* *Filter Bundle:* Repose (Standard)
+* link:../schemas/highly-efficient-record-processor.xsd[Schema]
+
+== Prerequisites & Postconditions
+=== Input
+==== Required headers
+This filter has no required headers.
+
+==== Required preceding filters
+This filter has no dependencies on other filters and can be placed wherever it is needed in the filter chain.
+
+[CAUTION]
```

Comment:
We may want to come up with something for the style guide with respect to admonitions. We could either prefer the paragraph style and only use the block style where necessary, or we could always use the block style.

In your case, a paragraph style admonition would work:
> CAUTION: 

You used a block style admonition even though the content does not appear to be complex.

See: http://asciidoctor.org/docs/user-manual/#admonition